lib/cli/index.js

```diff
@@ -75,8 +63,12 @@ function configureLogger(env) {
   }
 }
 
-// Options: Array cliArgs, Stream inputStream, Stream outputStream
+// Options: Array cliArgs, Stream inputStream, Stream outputStream, EventEmitter process
 module.exports = function(options) {
+  // `process` should be captured before we require any libraries which
+  // may use `process.exit` work arounds for async cleanup.
+  willInterruptProcess.capture(options.process || process);
```

Comment:
https://github.com/ember-cli/ember-cli/pull/6437 says that the actual problem was with `ora` which used by `console-ui`. So we need to capture `process` and `process.exit` before we require `console-ui`.